applications/callflow/src/module/cf_park.erl

```diff
@@ -534,13 +534,12 @@ get_parked_calls(AccountDb, AccountId) ->
 fetch_parked_calls(AccountDb, AccountId) ->
     case kz_datamgr:open_doc(AccountDb, ?DB_DOC_NAME) of
         {'error', 'not_found'} ->
-            Timestamp = calendar:datetime_to_gregorian_seconds(calendar:universal_time()),
             Generators = [fun(J) -> kz_doc:set_id(J, <<"parked_calls">>) end
                          ,fun(J) -> kz_doc:set_type(J, <<"parked_calls">>) end
                          ,fun(J) -> kz_doc:set_account_id(J, AccountId) end
                          ,fun(J) -> kz_doc:set_account_db(J, AccountDb) end
-                         ,fun(J) -> kz_doc:set_created(J, Timestamp) end
-                         ,fun(J) -> kz_doc:set_modified(J, Timestamp) end
+                         ,fun(J) -> kz_doc:set_created(J, kz_time:current_tstamp()) end
+                         ,fun(J) -> kz_doc:set_modified(J, kz_time:current_tstamp()) end
```

Comment:
since we're updating the fields with the same value, use `Timestamp` var instead and assign `kz_time:current_tstamp()`  above to `Timestamp`